src/app/personside/infotabs/oppfolging/OppfolgingDatoKomponent.tsx

```diff
@@ -82,8 +82,19 @@ function getDatoFeilmelding(fra: Date, til: Date) {
             </SkjemaelementFeilmelding>
         );
     }
-    if (!isValidDate(fra) || !isValidDate(til)) {
-        return <SkjemaelementFeilmelding>Du må velge gyldige datoer</SkjemaelementFeilmelding>;
+    if (!isValidDate(fra)) {
+        return (
+            <SkjemaelementFeilmelding>
+                Du må velge gyldig fra-dato. Gyldig datoformat er dd.mm.åååå
+            </SkjemaelementFeilmelding>
```

Comment:
Duplikat kode. Kan du ha bare et method her(fra og til kan sendes som string for ex.).